### DISCUSSION TITLE
mapping of old-to-new check IDs?

### DISCUSSION THREAD

**Discussion by bobh0303** [fonttools/fontbakery#4875] [Q&A] [answered]
With the elimination of reverse-domain naming and the shuffling of checks into vendor-specific profiles nearly done, is there — or could someone generate — a reference mapping from old check ID to new check ID? Manually checking and changing older profiles and support tools for the 300+ check IDs is painful.

**Comment by felipesanches** [comment, marked as answer]
yes, I'll provide that next week.

**Reply by bobh0303** [reply]
That'd be awesome. Thanks.

**Reply by felipesanches** [reply]
It took me "a bit" more than a week to do it, but here it is ;-) At #4929 I introduced a python dictionary that maps the old check IDs to the new ones. Take a look at `Lib/fontbakery/legacy_checkids.py`

**Reply by bobh0303** [reply]
Thanks SO much for this. This, along with the CHANGELOG, have saved me hours of research.

**Reply by felipesanches** [reply]
I'm glad these things are helpful. That's why I've put the effort, in the first place.
